src/main/java/br/com/caelum/clines/shared/exceptions/LocationNotFoundException.java

```diff
@@ -1,5 +1,9 @@
 package br.com.caelum.clines.shared.exceptions;
 
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
Given that we have a `GlobalExceptionHandler`, it makes sense keep this annotation?

Reply:
The test does not work without it

Reply:
(We will review this code, yesterday we hadn't Fernando's code)

Reply:
The test still does not works without it

Reply:
Wrong exception type. Fixed now.